server/entity/splash_potion.go

```diff
@@ -0,0 +1,245 @@
+package entity
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/entity/effect"
+	"github.com/df-mc/dragonfly/server/entity/physics"
+	"github.com/df-mc/dragonfly/server/entity/physics/trace"
+	"github.com/df-mc/dragonfly/server/internal/nbtconv"
+	"github.com/df-mc/dragonfly/server/item/potion"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/particle"
+	"github.com/df-mc/dragonfly/server/world/sound"
+	"github.com/go-gl/mathgl/mgl64"
+	"image/color"
+	"math"
+	"time"
+)
+
+// SplashPotion is an item that grants effects when thrown.
+type SplashPotion struct {
+	transform
+	yaw, pitch float64
+
+	ticksLived int
+
+	closeNextTick bool
+
+	owner world.Entity
+
+	variant potion.Potion
+
+	c *ProjectileComputer
+}
+
+// NewSplashPotion ...
+func NewSplashPotion(pos mgl64.Vec3, yaw, pitch float64, owner world.Entity) *SplashPotion {
+	s := &SplashPotion{
+		yaw:   yaw,
+		pitch: pitch,
+		c: &ProjectileComputer{&MovementComputer{
+			Gravity:           0.05,
+			Drag:              0.01,
+			DragBeforeGravity: true,
+		}},
+		owner: owner,
+	}
+	s.transform = newTransform(s, pos)
+
+	return s
+}
+
+// Name ...
+func (s *SplashPotion) Name() string {
+	return "Splash Variant"
```

Comment:
```suggestion
	return "Splash Potion"
```

Accidental refactor?

Reply:
Woops, yeah